source/transactions/tests/commit.json

```diff
@@ -350,6 +350,7 @@
     },
     {
       "description": "write concern error on commit",
+      "skipReason": "SERVER-37458 Mongos does not yet support writeConcern on commit",
```

Comment:
Is this merely a comment or are drivers expected to actually skip any and all tests that include a `skipReason` field? If the latter, that may be worth specifying in the test readme.

Reply:
It is already described:
```
  - ``skipReason``: Optional, string describing why this test should be
    skipped.
```

Reply:
Also added:
```
#. If the``skipReason`` field is present, skip this test completely.
```